### PR TITLE
Add elo rating

### PR DESCRIPTION
**Pull request by Mervelin**

### CODE REVIEW

File: api/db/migrate/20181005100009_add_elo_rating_to_player.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddEloRatingToPlayer < ActiveRecord::Migration[5.2]
+  def change
+    add_column :players, :elo_rating, :integer, null: false, default: 1200
```
Comment (mhib) [inline api/db/migrate/20181005100009_add_elo_rating_to_player.rb]: I'd consider adding separate defender and attacker rating.

Comment (wojtowicz) [reply api/db/migrate/20181005100009_add_elo_rating_to_player.rb]: @mhib could you explain more?

**Comment by wojtowicz** [comment]
Hi @Mervelin. Please add some specs.

**Comment by wojtowicz** [comment]
Hi @Mervelin. Please rebase

**Pull request merged**